ansible/roles/riakcs/README.md

```diff
@@ -8,7 +8,9 @@ For a system overview look [here](https://docs.google.com/document/d/1F5KjtyXmvG
 
 2. Run `deploy_proxy` to update Nginx with the new configs to allow webworkers to communicate with RiakCS cluster
 
-3. Run `deploy_riakcs` to deploy the Riak CS cluster
+3. Optional: run `deploy_stack --tags=datadog --limit=riakcs` to setup the datadog monitoring agent.
+
+4. Run `deploy_riakcs` to deploy the Riak CS cluster
```

Comment:
I agree, that would be nice, but it would be a larger structural change: the datadog tasks would need to be moved after `deploy_riakcs`, which is included by `deploy_db`. Currently the datadog tasks are part of `deploy_common`, which happens before anything else.

Actually, it might make a lot of sense to move the datadog tasks to the end of `deploy_stack` since it would be better to configure the services that datadog will be monitoring before setting up the monitoring service itself so we don't end up with service outage alerts during initial configuration of new servers.